src/groups/mqb/mqba/mqba_sessionnegotiator.cpp

```diff
@@ -729,6 +739,24 @@ void SessionNegotiator::createSession(bsl::ostream& errorDescription,
             : negoMsg.brokerResponse().brokerIdentity();
     const mqbcfg::AppConfig& brkrCfg = mqbcfg::BrokerConfig::get();
 
+    // reading maxMissedHeartbeat from config
+    //
+    // ConnectionType::e_ADMIN                                 - 0
+    //
+    // ConnectionType::e_CLIENT:
+    //      bmqp_ctrlmsg::ClientType::E_TCPCLIENT              - client()
+    //      bmqp_ctrlmsg::ClientType::E_TCPBROKER              -
+    //      downstreamBroker()
+    //
+    // ConnectionType::e_CLUSTER_PROXY:
+    // ConnectionType::e_CLUSTER_MEMBER:
+    //      selfNodeId() == mqbnet::Cluster::k_INVALID_NODE_ID -
+    //      upstreamBroker() selfNodeId() != mqbnet::Cluster::k_INVALID_NODE_ID
+    //      - clusterPeer()
```

Comment:
The layout of comments is broken it seems. We can also remove this section because we can understand it from below